### PR TITLE
Add new OpenTelemetry JVM UI details

### PR DESCRIPTION
**Pull request by rhetoric101**
This is work for Jira doc-7771. We have some new features on the OTEL JVM page.

Reviewer: Here's what I changed:

* Inserted new text to augment the existing text in the JVM page.
* I segmented some sub-tasks using new collapsers. See if those collapsers make sense.
* The final collapser has some detailed instructions. Is this clear?

### CODE REVIEW

**Comment by rhetoric101** [comment]
Temporarily closing to see if reopening it will fix the build.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 1]
```diff
@@ -262,15 +262,63 @@ For more details, see [External services](/docs/apm/apm-ui-pages/monitoring/exte
...
+
+  The solution for now (until a better solution is sorted out in the OpenTelemetry metric specification) is to use the View API to indicate that async up down counters should be aggregated using last value aggregation instead of the default sum aggregation. This results in JVM memory usage being exported as gauge data, which is required for a useful experience in New Relic.
+
+  The way you configure the View API varies based on whether or not you’re using the OpenTelemetry Java agent:
 
-  * Requires a unique `service.instance.id` attribute for rendering the list of JVMs
-  * `Service.instance.id` is an OpenTelemetry resource attribute
+  * If you're not using the OpenTelemetry Java agent, review this simple [example](https://github.com/newrelic/newrelic-opentelemetry-examples/pull/89/files#diff-da355ef6d1092534a55829e95160ab8468884bdd521f9018feeaaa66aea6ac5bR82-R86) that shows how to register a view when configuring your `SdkMeterProvider`.
```
Comment (barbnewrelic) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:318]: This link goes to a diff. Is that intentional?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx [hunk 2]
```diff
@@ -262,15 +262,63 @@ For more details, see [External services](/docs/apm/apm-ui-pages/monitoring/exte
...
+
+    * A unique `service.instance.id` attribute for rendering the list of JVMs
+    * An OpenTelemetry resource attribute `Service.instance.id`.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="metric-types"
+    title="Gauges versus counters"
+  >
+  Starting in OpenTelemetry Java agent 1.10.0, JVM memory usage switched from being collected as an async gauge to an async up down counter. This has implications on the exported data. Gauges and counters export differently:
```
Comment (barbnewrelic) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:305]: Throughout this section, is "up down" ok or should it be "up-down"? I'm not sure based on the context, so I'll leave that decision to you.

**Review by barbnewrelic — DISMISSED** [review]
I am not familiar with this content, so the 2 questions I noted in my review may be irrelevant. Feel free to ignore. I trust your judgment!

**Review by barbnewrelic — APPROVED** [review]
It looks good to me. Thanks for all your hard work on this!

**Review by barbnewrelic — APPROVED** [review]
I reviewed this doc in your Gatsby preview one last time and it looks really good. I appreciate how thorough you are with this doc. A lot of info, but it never feels too overwhelming because it's structured into practical workflows, examples, screenshots, tables, and collapsers. Congrats!

**Pull request merged**